interface-spec.md

```diff
@@ -38,9 +43,7 @@ TODO: to what extent should we use typed signatures (`.equals(Node other)`) vers
 
 **Properties:**
 
-- `String .value` blank node name as a string (example: `_:blank3`)
-
-TODO: Does the value always start with an underscore?
+- `String .value` blank node name as a string without leading `_:` (example: `blank3`)
```

Comment:
-1

I don't see the value of removing `_:` from the value -- that is actually part of the identifier for the blank node. Furthermore, blank node identifiers are not part of the RDF abstract syntax -- the presumption that they start with `_:` is only based on the fact that nearly all concrete syntaxes use this notation. While unlikely, it's *possible* that another syntax would not use that form.

If the actual value of a blank node identifier is not going to be found via the `.value` property, then I think there should be a very good argument for taking that approach. IMO, this approach would be unexpected.